### PR TITLE
Docker environ

### PR DESCRIPTION
**Pull request by alastair**

### CODE REVIEW

File: environment.ctmpl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+EMAIL_BACKEND={{template "KEY" "email_backend"}}
+WEB_SECRET_KEY={{template "KEY" "web_secret_key"}}
+{{if service "brainzbot-postgresql"}}
+{{with index (service "brainzbot-postgresql") 0}}
+STORAGE_URL="postgresql://postgres:{{template "KEY" "storage_password"}}@{{.Address}}:{{.Port}}/{{template "KEY" "storage_database"}}"
+{{end}}
+{{end}}
+
+{{if service "prod.brainzbot-redis"}}
+{{with index (service "listenbrainz-redis") 0}}
```
Comment (amCap1712) [inline environment.ctmpl:15]: service "prod.brainzbot-redis"

Comment (alastair) [reply environment.ctmpl:15]: oops

File: docker/services/uwsgi/uwsgi.ini [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+enable-threads = true
+processes = 30
+
+log-x-forwarded-for=true
+disable-logging = true
+; quit uwsgi if the python app fails to load
+need-app = true
+; when uwsgi gets a sighup, quit completely and let runit restart us
+exit-on-reload = true
```
Comment (alastair) [inline docker/services/uwsgi/uwsgi.ini:15]: this file wasn't finished, it's a mix between LB and the current botbot interface

**Review by amCap1712 — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**